src/components/SingleBlog/index.test.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { BrowserRouter as Router } from "react-router-dom";
+import { render } from "react-dom";
+import { act } from "react-dom/test-utils";
+import SingleBlog from "./index";
+
+let container;
+
+beforeEach(() => {
+  container = document.createElement("div");
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  document.body.removeChild(container);
+  container = null;
+});
+
+describe("Single blog", () => {
+  it("Renders without crashing", async () => {
+    const mockData = {
```

Comment:
probably should've fixed in the other PR too but

MOCK_DATA and put it below line 19 or at line 18, either works (in between describe+it or above describe)

Reply:
okay, i will move here and in separate PR will move in the other one